### PR TITLE
Add Usage guide for Docker Containerizer with Calico

### PR DESCRIPTION
**Pull request by TrimBiggs**

### CODE REVIEW

**Comment by djosborne** [comment]
I think we need a new title:
"Calico-Mesos Usage Guide for the Docker Containerizer"?

**Comment by TrimBiggs** [comment]
^ Done

File: docs/UsageGuideDockerContainerizer.md [hunk 1]
```diff
@@ -16,43 +16,51 @@ containerizer.  This guide covers:
 -  Creating a Docker network and managing network policy
 -  Launching a container
 
-## Set up Docker multi-host networking
+## Prerequisites
 
 To utilize the Docker libnetwork plugin feature of Docker, it is necessary to 
 run with a Docker version 1.9 or higher, and to configure Docker to use a
```
Comment (djosborne) [inline docs/UsageGuideDockerContainerizer.md]: Want to swap this so its the first thing in the prerequisites section?

File: docs/UsageGuideDockerContainerizer.md [hunk 2]
```diff
@@ -16,43 +16,51 @@ containerizer.  This guide covers:
...
-cluster store.  We'll set up an etcd datastore which will be used by Docker and
-Calico.
+cluster store.  
 
-#### Install etcd
+You can easily configure a mesos cluster by running through
```
Comment (djosborne) [inline docs/UsageGuideDockerContainerizer.md]: Want to swap this so its the first thing in the prerequisites section?

File: docs/UsageGuideDockerContainerizer.md [hunk 3]
```diff
@@ -65,52 +73,7 @@ chmod a+x calicoctl
...
-+----------------+---------+
-+--------------------------+---------+
-|        IPv6 CIDR         | Options |
-+--------------------------+---------+
-| fd80:24e2:f998:72d6::/64 |         |
-+--------------------------+---------+
-```
+Be sure to set the ETCD_AUTHORITY to the correct `IP:Port` for your etcd cluster.
 
 ## Creating a Docker network and managing network policy
```
Comment (djosborne) [inline docs/UsageGuideDockerContainerizer.md]: I think the first line in this section should read: "Before we can start launching tasks, we must first create a docker network with Calico."

**Comment by djosborne** [comment]
IMO the docker network commands look better with `=` signs:
` docker network create --driver=calico --ipam-driver=calico applications`

File: docs/UsageGuideDockerContainerizer.md [hunk 4]
```diff
@@ -194,16 +158,16 @@ Outbound rules:
...
 
 With your networks configured, it is trivial to launch a Docker container 
 through Mesos using the standard Marathon UI and API.
 
 #### Launching a container through the UI
 
-Through the UI, launch a Docker task as normal.  Select an arbitrary(*) network
-(Bridge or Host), and then provide the following additional parameter 
-(under the Docker options)
+Through the Marathon UI at `MARATHON_IP:8080`, launch a Docker task as normal.
```
Comment (djosborne) [inline docs/UsageGuideDockerContainerizer.md]: English is a little off here. reversing the statements sounds better: Launch a Docker task as normal through the Marathon UI at `MARATHON_IP:8080`

File: docs/UsageGuideDockerContainerizer.md [hunk 5]
```diff
@@ -243,5 +207,43 @@ the net parameter in the request.  For example:
 }
```
Comment (djosborne) [inline docs/UsageGuideDockerContainerizer.md]: As someone who refers to these guides regularly, Its a pain to dig up the curl command that kicks these off. Would be nice if we added it:
`curl -X PUT -H "Content-Type: application/json" http://localhost:8080/v2/groups/calico-apps @app.json`.

**Comment by djosborne** [comment]
Few things, then LGTM for now. After this gets merged, next time I have train time I think I'll do a word-by-word readthrough might open up a new PR with some english fixes.

**Comment by TrimBiggs** [comment]
The change wasn't very elaborate and could use some additional work sometime soon

File: docs/UsageGuideDockerContainerizer.md (hunk 5 above)
Comment (TrimBiggs) [inline docs/UsageGuideDockerContainerizer.md]: Done

**Pull request merged**